src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+
+  --hover-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
```

Comment:
CSS: You have fixed the size of the `.logo` class. It's not recommended to fix the size of containers unless there is a specific requirement. Let the content size dictate it to avoid overflow or accidental scroll bar.